tests/phpunit/src/Commands/Remote/AliasesListCommandTest.php

```diff
@@ -45,7 +45,7 @@ public function testRemoteAliasesListCommand(): void {
     $this->prophet->checkPredictions();
     $output = $this->getDisplay();
 
-    $this->assertStringContainsString('| Sample application 1 | devcloud2.dev | 24-a47ac10b-58cc-4372-a567-0e02b2c3d470 |', $output);
+    $this->assertStringContainsString('| Sample application 1 | devcloud2.dev     | 24-a47ac10b-58cc-4372-a567-0e02b2c3d470 |', $output);
```

Comment:
Checking the string in this way feels like a code-smell, but also doesn't feel like something worth spending time to try to do more flexibly